async-nats/src/lib.rs

```diff
@@ -1068,17 +1077,17 @@ pub struct Message {
     pub reply: Option<String>,
     pub payload: Bytes,
     pub headers: Option<HeaderMap>,
+    status: Option<NonZeroU16>,
+    description: Option<String>,
 }
 
 impl Message {
     fn is_no_responders(&self) -> bool {
```

Comment:
Make this an extension trait?

```suggestion
    fn is_no_responders(&self) -> bool {
```

Reply:
it is a private function now.